sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreConf.scala

```diff
@@ -117,6 +117,10 @@ class StateStoreConf(
   val reportSnapshotUploadLag: Boolean =
     sqlConf.stateStoreCoordinatorReportSnapshotUploadLag
 
+  /** Whether to force snapshot when a state store is lagging behind in snapshot uploads. */
+  val stateStoreAutoSnapshotForLaggingStoresEnabled: Boolean =
```

Comment:
nit: no need for `stateStore` prefix in the val name. The name of the class is already `StateStoreConf`. Also fix val name based on config name suggestion above